app/views/application/_action_bar.html.erb

```diff
@@ -1,13 +1,14 @@
 <div class="absolute bottom-12 left-0 right-0 flex flex-row justify-center">
-  <div class="action-bar" data-controller="action-bar">
-	<button data-action="click->action-bar#deleteTasks">
-	  <%= heroicon "trash" %>
-	  <span data-action-bar-target="deleteButtonLabel"></span>
-	</button>
-	
-	<button>
-	  <%= heroicon "arrow-circle-right" %>
-	  <span>Move (0)</span>
-	</button>
+  <div class="action-bar" data-action-bar-target="bar">
+    <%= button_to(batch_destroy_tasks_path, method: :delete, data: { 'action': 'action-bar#deleteTasks' }) do %>
```

Comment:
Could probably get rid of this action completely and let the `check` actions populate the `taskIds` target as they change. /shrug